app/controllers/flights_controller.rb

```diff
@@ -0,0 +1,5 @@
+class FlightsController < ApplicationController
+  def index
+    @flights = Flight.with_airline_and_passengers
+  end
+end
```

Comment:
Ok, your solution for this US is over-engineered. You are not making use of your associations to get to the data that you want. There is no reason to create a model method for this. All the information is just one or two dots (`.`) away.

For example, a flight's airline name can be gotten from `flight.airline.name`. The other information like the passengers can be retrieved just as easily.